src/renderer/init.js

```diff
@@ -61,6 +65,20 @@ async function init() {
     connect,
   });
 
+  const oldUserId = global.localStorage.getItem("userId");
+  if (oldUserId) {
+    global.localStorage.removeItem("userId");
+  }
+
+  let user = await getKey("app", "user");
+  if (!user) {
+    user = { id: uuid() };
+    setKey("app", "user", user);
```

Comment:
i think an await is needed on setKey

Reply:
https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/storage.js#L59 looks ok